source/_docs/installation/windows.markdown

```diff
@@ -15,6 +15,8 @@ To run Home Assistant on Microsoft Windows installation you need to install Pyth
 There may be alpha or beta releases of Python listed on that download page (marked by the letters `a` or `b` in the version number. Do not use these versions.
 </p>
 
+If you use you machine for something other than Home Assistant you should install it in a [python virtual environment](#Installing_in_a_python_virtual_environment)
+
```

Comment:
Please fix the typo and add a period to the end of the sentence. It's preferred to use a capital `P` for Python.